src/workerd/api/actor-state.c++

```diff
@@ -370,7 +370,7 @@ kj::Maybe<DurableObjectStorageOperations::CompiledListOptions> DurableObjectStor
         // Calculate the first key that sorts after all keys with the given prefix.
         kj::Vector<char> keyAfterPrefix(prefix.size());
         keyAfterPrefix.addAll(prefix);
-        while (!keyAfterPrefix.empty() && (byte)keyAfterPrefix.back() == 0xff) {
+        while (!keyAfterPrefix.empty() && static_cast<byte>(keyAfterPrefix.back()) == 0xff) {
```

Comment:
Honestly, I prefer the old style here, the new approach is more verbose and makes code harder for humans to read IMHO.